pkg/flow/states/action.go

```diff
@@ -198,27 +200,25 @@ func (logic *actionLogic) processActionResults(ctx context.Context, children []*
 	if results.ActionID != id {
 		return nil, derrors.NewInternalError(errors.New("incorrect child action ID"))
 	}
-
-	logic.Log(ctx, "Child '%s' returned.", id)
+	logic.Log(ctx, util.Info, "Function %s returned, id: %s.", logic.label(), id)
```

Comment:
I call them "child" here because when we say "function" we usually mean a knative service, but this could also be a subflow.